File: Cpp/1598_Crawler_Log_Folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int result = 0;
        for(auto& log : logs){
            if(log == "../") {
                result = max(0, result - 1);
            } else if(log != "./"){
                ++result;
            }
        }
        
        return result;
    }
};

File: Kotlin/1598_Crawler_Log_Folder.kt
class Solution {
    fun minOperations(logs: Array<String>): Int {
        var result = 0
        for(log in logs){
            if(log.equals("../")) {
                result = maxOf(0, result - 1)
            } else if(!log.equals("./")){
                ++result
            }
        }
        
        return result
    }
}